build/make-initrd.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
 rm -rf root
-mkdir -p root/lib
+mkdir -p root/lib root/lib64
 
 cp ../src/init ./root
 cp busybox ./root
+cp iptables ./root
+cp libm.so.6 ./root/lib64/
```

Comment:
libm.so.6 is not shown in the ldd? see the ldd in the following lines in this file.